src/style.css

```
@@ -1 +1,57 @@
-/* Styles go here */
+body,
+.nav__list {
+  margin: 0;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
+  padding: 0;
+}
+
+.link {
+  display: flex;
+  align-items: center;
+  color: #000;
+  text-decoration: none;
+  text-transform: uppercase;
+  position: relative;
+  height: 60px;
+  margin-left: 20px;
+}
+
+.link:hover {
+  color: rgba(0, 172, 220, 1);
+}
+
+.nav__item:first-child .link {
+  margin-left: 0;
+}
```

do not combine selectors. why not
```suggestion
.link:first-child {
  margin-left: 0;
}
```

---

@ihor-jpeg When I don't combine the selectors all the links are selected, I think is because they're all first children of the li(list) tag